hack/ci-e2e-test.sh

```diff
@@ -19,6 +32,10 @@ ssh-keygen -f $HOME/.ssh/id_rsa -P ''
 echo "Creating environment at cloud provider."
 make -C test/tools/integration apply
 
+# Build binaries
+echo "Building machine-controller and webhook"
+make machine-controller webhook
```

Comment:
We have to explicitly build them in the case of prow, with circleci we passed them around via a cache but we don't have that anymore